### PR TITLE
chore: add Rahulbaran

### PR DESCRIPTION
**Pull request by Rahulbaran**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/915"></a>

### CODE REVIEW

File: public/data/Rahulbaran.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Rahul Kumar",
+  "type": "personal",
+  "bio": "Web developer | Love working with CSS, JavaScript and Flask",
```
Comment (Panquesito7) [inline public/data/Rahulbaran.json:4]:
```suggestion
  "bio": "Web developer | Love working with CSS, JavaScript, and Flask",
```

Comment (Rahulbaran) [reply public/data/Rahulbaran.json:4]: Should I delete this PR and make another one after removing the **comma**???

Comment (krishguptadev) [reply public/data/Rahulbaran.json:4]: You can push to the same branch and it will work!

File: public/data/Rahulbaran.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Rahul9422dev/",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/Rahulbaran/",
```
Comment (krishguptadev) [inline public/data/Rahulbaran.json:19]:
```suggestion
      "url": "https://www.linkedin.com/in/Rahulbaran",
```

File: public/data/Rahulbaran.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "avatar": "https://github.com/Rahulbaran.png",
+  "links": [
+    {
+      "name": "Checkout my GitHub Repositories",
+      "url": "https://github.com/Rahulbaran",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Rahul9422dev/",
```
Comment (krishguptadev) [inline public/data/Rahulbaran.json:14]:
```suggestion
      "url": "https://twitter.com/Rahul9422dev",
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by syedsohan — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Pull request merged**